src/main.js

```diff
@@ -125,15 +125,32 @@ function removePoster() {
    showSavedPosters();
 }
 
-// extensions
-
+function getRandomIndex(array) {
+  return array[Math.floor(Math.random() * array.length)];
+}
 
+// extension 1:
 function verifyForm(url, title, quote) {
   if (url === "" || title === '' || quote === '') {
     return false;
   }
 }
-// (we've provided one for you to get you started)!
-function getRandomIndex(array) {
-  return array[Math.floor(Math.random() * array.length)];
+
+//extension 2:
+displayTitle.addEventListener('click', replace;
```

Comment:
Looks like you are missing a closing paren on this function

Reply:
good catch!